doc/whats_new/v1.3.rst

```diff
@@ -55,11 +66,13 @@ Changes impacting all modules
   - :class:`preprocessing.RobustScaler`
   - :class:`preprocessing.SplineTransformer`
   - :class:`preprocessing.StandardScaler`
+  - :class:`random_projection.GaussianRandomProjection`
+  - :class:`random_projection.SparseRandomProjection`
 
   The `NotFittedError` displays an informative message asking to fit the instance
   with the appropriate arguments.
 
-  :pr:`25294` by :user:`John Pangas <jpangas>` and :pr:`25291`, :pr:`25367` by
+  :pr:`25294` , :pr:`25308` by :user:`John Pangas <jpangas>` and :pr:`25291`, :pr:`25367` by
   :user:`Rahil Parikh <rprkh>` and :pr:`25402` by :user:`Alex Buzenet <albuzenet>`.
```

Comment:
I was proposing changing this part to look like the text below for easier readability. @glemaitre , is there a conventional style of arranging these sentences?
:pr:`25294` , :pr:`25308` by :user:`John Pangas <jpangas>` ,
:pr:`25291`, :pr:`25367` by :user:`Rahil Parikh <rprkh>` ,
and :pr:`25402` by :user:`Alex Buzenet <albuzenet>`.

Reply:
Let's put all the pr numbers and then the list of contributors. We don't really care about the association but rather the original PR and provide authorship credits.